src/views/Namespace.vue

```diff
@@ -113,6 +113,9 @@ export default {
 
         ]).catch(this.errHandler).then(() => {
           this.loaded = true
+          this.$ComposeAPI.settingsGet({ key: 'panel.namespace-switcher' }).then(({ value = false }) => {
+            this.showNSSideBar = value && !!window.SHOW_NAMESPACE_PANEL
```

Comment:
`window.SHOW_NAMESPACE_PANEL` will be deprecated as soon as this goes up, so no need for it